.github/workflows/financialconnections_pull_request.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           ref: ${{ github.base_ref }}
       - uses: ./.github/actions/stripe_setup
       - name: Build base branch
-        run: ./gradlew :financial-connections-example:assembleRelease && mv financial-connections-example/build/outputs/apk/release/financial-connections-example-release-unsigned.apk financial-connections-example/build/outputs/apk/release/financial-connections-example-release-base.apk
```

Comment:
These will fail on this PR because master branch (which this also builds) doesn't contain the signed version, and will fail to move. I checked all of the errors, and made sure they wouldn't happen again after this PR is merged though.